todo_app/tests/test_integration.py

```diff
@@ -0,0 +1,83 @@
+from logging import log
+import todo_app
+from todo_app.data import itemstatus
+from todo_app.data.itemstatus import ItemStatus
+from todo_app.model.viewModel import ViewModel
+from todo_app.data.item import Item
+import pytest
+from dotenv import load_dotenv, find_dotenv
+
+from unittest.mock import patch, Mock
+import todo_app.app
+
+@pytest.fixture
+def client():
+ # Use our test integration config instead of the 'real' version
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+ # Create the new app.
+    test_app = todo_app.app.create_app()
+    test_app.secret_key = 'super secret key'
+    test_app.config['SESSION_TYPE'] = 'filesystem'
```

Comment:
If you fix up the comments above re setting the Secret Key as normal, then we could remove these